### ISSUE TITLE
More Options: Hiding options. 

### ISSUE THREAD

**Issue by bph** [WordPress/gutenberg#28560]
## Description
When I opened the More Options menu on a paragraph, I wasn't able to see the "Remove Block" option as it was on the bottom of the little window, out of the frame. The window does not have a visible scroll bar. User who don't know that the remove block option is below the visible part, won't scroll down to find it. 

## Step-by-step reproduction instructions

1. Create a paragraph
2. Open the 3-dot menu

## Expected behaviour
A visual indication that there are more option below the visible frame. 

## Screenshots or screen recording (optional)
Here is a little video https://recordit.co/dkMNIKLEo5

## WordPress information
- WordPress version: 5.6
- Gutenberg version: (9.8.2 from master)

**Comment by talldan** [comment]
> The window does not have a visible scroll bar.

@bph It looks like you're on a mac where this is an OS setting under System Preferences > General > Show scroll bars. I'm not sure what can be done from a browser point of view if that's set to something other than 'Always'.

There might be some design magic we can work to improve this, so I'll add the `Needs design feedback` label.

Historically, I think in the Link UI we tried a little gradient at the bottom of the popover when it could be scrolled, but looks like that has since been removed.
